quizcon/templates/main/question_form_edit.html

```diff
@@ -0,0 +1,88 @@
+{% extends 'base.html' %}
+{% load bootstrap4 %}
+
+{% block title %}Edit Question{% endblock %}
+
+{% block pagetitle %}
+    Edit Question
+{% endblock %}
+
+{% block content %}
+<div class="container container-content">
+<!-- Edit Question -->
+	<h1>
+		Edit Question
+	</h1>
+	<nav aria-label="breadcrumb">
+		<ol class="breadcrumb">
+			<li class="breadcrumb-item"><a href="my-quizzes.html">My Quizzes</a></li>
+			<li class="breadcrumb-item"><a href="edit-quiz.html">{{quiz.title}}</a></li>
+			<li class="breadcrumb-item active" aria-current="page">Question 1</li>
+		</ol>
+	</nav>
+	<form action="." method="post" v-on:submit="onSubmit">{% csrf_token %}
+		<div class="row mb-3">
+			<label for="thetextarea-q" class="col-sm-2 col-form-label">Question</label>
+			<div class="col-sm-10">
+				<textarea class="form-control" id="edit-text" name="text">{{object.text}}</textarea>
+			</div>
+		</div>
+		<div class="row mb-3">
+			<label for="thetextarea-a" class="col-sm-2 col-form-label">Answer A</label>
+			<div class="col-sm-10">
+				<textarea class="form-control" id="thetextarea-a">Jupiter</textarea>
```

Comment:
Looks like the old boilerplate was left here in the form. Can you update to render the stored answer text?